README.md

```diff
@@ -188,6 +188,7 @@ The teacherservices.cloud domain is created in route53 and owned by infra-ops. S
 - [Onboarding form template](documentation/onboard-form-template.md)
 - [Kubernetes cluster Public IPs](documentation/public-ips.md)
 - [Production checklist](documentation/production-checklist.md)
+- [Maintenance page](documentation/maintenance-page.md)
```

Comment:
please reconsider reconstructing the grammar near Front door sends traffic to the ingress on Kubernetes. By default to the teacherservice.cloud internal ingress due to not being clear